Dump20190526.sql

```diff
@@ -0,0 +1,37 @@
+-- MySQL dump 10.13  Distrib 5.7.24, for osx10.13 (x86_64)
```

Comment:
What is the purpose of this file?

Reply:
This id dump of database. I'm using database in this task.